about.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>About Gamecon</title>
+</head>
+
+<body>
+    <header>
+        <div id="topnav">
+            <a href=""><img src="./images/pngkey.com-facebook-logo-png-white-1532072.png" alt="Facebook"
+                    class="social"></a>
+            <a href=""><img src="./images/pngkey.com-twitter-logo-png-transparent-111779.png" alt="Twitter"
+                    class="social"></a>
+            <ul id="topnavList">
+                <li>English</li>
+                <li>Home</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <nav>
+            <img src="./images/menu.png" alt="Menu button" id="menuOpen">
+            <a href="./index.html"><img src="./images/Gamecon2023.png" alt="Gamecon 2023" id="gameconHome"></a>
+            <ul id="menuList">
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="#logoDiv">Logo</a></li>
+                <li><a href="#history">History</a></li>
+                <li><a href="#sponser">Sponsers</a></li>
+            </ul>
+            <ul id="deskList">
+                <li><a href="./about.html" id="aBtn">About</a></li>
+                <li><a href="#logoDiv" class="bBtn">Logo</a></li>
+                <li><a href="#history" class="bBtn">History</a></li>
+                <li><a href="#sponser" class="bBtn">Sponsers</a></li>
+                <li><button type="button" id="gamefairBtn">Campaign</button></li>
+            </ul>
```

Comment:
- [ ] For the desktop screens, once you are on the `About` page, your `About` nav item should be turned into `Home` instead, kindly fix that.

 ![image](https://github.com/SnakyMz/Gamecon/assets/102612396/e0cc7dc4-f879-49a9-b923-895757a403ba)

Reply:
Thanks I was confused since the template had 'About' in its about page
![Screenshot 2023-06-30 013035](https://github.com/SnakyMz/Gamecon/assets/115364362/1c6e0a71-10e8-4758-90f4-f04803f86b40)